docs/quickstart.rst

```diff
@@ -207,19 +211,19 @@ Now that we have Kafka and Zookeeper up and running, we can deploy some of the o
 
   .. sourcecode:: bash
 
-    docker logs schema-registry
+    $ docker logs schema-registry
 
   For the next step, we'll publish data to a new topic that will leverage the Schema Registry. For the sake of simplicity, we'll launch a second Schema Registry container in interactive mode, and then execute our ``kafka-avro-console-producer`` utility from there.
 
   .. sourcecode:: bash
 
-    docker run -it --net=host --rm confluentinc/cp-schema-registry:3.3.0-SNAPSHOT bash
+    $ docker run -it --net=host --rm confluentinc/cp-schema-registry:3.3.0-SNAPSHOT bash
 
   Direct the utility at the local Kafka cluster, tell it to write to the topic ``bar``, read each line of input as an Avro message, validate the schema against the Schema Registry at the specified URL, and finally indicate the format of the data.
 
   .. sourcecode:: bash
 
-    /usr/bin/kafka-avro-console-producer \
+    # /usr/bin/kafka-avro-console-producer \
```

Comment:
Unfortunately, it's not, at least not with our current images. When we run a shell (`bash` here) it will run it as root. I skipped the most "frightening" part of the prompt that will actually be `root@confluent:/#` so that we can be both accurate about what will happen and not too distracting at this quickstart.